src/main/java/no/nav/fo/veilarbdialog/kvp/KvpServiceConfig.java

```diff
@@ -0,0 +1,29 @@
+package no.nav.fo.veilarbdialog.kvp;
+
+import no.nav.common.rest.client.RestClient;
+import no.nav.common.token_client.client.AzureAdMachineToMachineTokenClient;
+import okhttp3.OkHttpClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import static no.nav.common.utils.EnvironmentUtils.isProduction;
+
+@Configuration
+public class KvpServiceConfig {
+
+    @Value("${application.veilarboppfolging.api.url}")
+    private String baseUrl;
+
+    @Bean
+    public KvpService kvpService(AzureAdMachineToMachineTokenClient tokenClient) {
+        String tokenScope = String.format(
+                "api://%s-fss.pto.veilarboppfolging/.default",
```

Comment:
Det hadde vært litt mer fleksibelt å sette scope som en property, i likhet med URLen. F.eks `application.veilarboppfolging.api.scope`

Reply:
@AlexGaard godkjenn bare: https://github.com/navikt/pto-config/pull/37

Reply:
aha... skjønner..skal fikse..

Reply:
@AlexGaard er det bedre nå?

Reply:
Ser bra ut 😄 
Ikke at det burde utføres i denne oppgaven, men på sikt @holymaloney så ville jeg anbefalt å flytte all config som dere bruker inn i repoet deres, så får dere bedre kontroll.

Reply:
Btw, det mangler å sette `application.veilarboppfolging.api.scope` for testene også slik at de går igjennom

Reply:
Fikset